### DISCUSSION TITLE
is not a valid process webhook job class.

### DISCUSSION THREAD

**Discussion by DanielCriollo** [spatie/laravel-webhook-client#122] [General]
Hello, can someone help me to fix this error,

Spatie \ WebhookClient \ Exceptions \ InvalidConfig: `` is not a valid process webhook job class. A valid class should implement `Spatie \ WebhookClient \ ProcessWebhookJob`. in file C: \ laragon \ www \ example-app \ vendor \ spatie \ laravel-webhook-client \ src \ Exceptions \ InvalidConfig.php on line 43

Just following the documentation steps, it didn't work

**Comment by andrewlindemulder** [comment]
you need to set the "process_webhook_job" to a valid class, like this for example: App\Jobs\ProcessWebhooks::class
